cupy/core/reduction.pxi

```diff
@@ -10,7 +10,7 @@ from cupy.cuda import compiler
 from cupy import util
 
 
-cpdef _get_simple_reduction_kernel(
+cpdef _create_reduction_kernel(
```

Comment:
This function returns `cupy.cuda.function.Function`, not `ReductionKernel`. How about `_create_reduction_function` or `_create_reduction`?
BTW, why not writing the type of return value explicitly for readability?

Reply:
`_create_reduction_function` looks good to me.
In that case, I want to rename other functions similarly.
For example,
* `_AbstructReductionKernel._get_kernel()` -> `_get_function()`
* `_ReductionKernel_get_cached_kernel` -> `_Reductionkernel_get_cached_function`
* etc.

Do you think it's ok?

Reply:
I agree with explicitly typing the return value.

Reply:
Yes, `_get_function` looks better to me too.

Reply:
Fixed in fd1d645e91bbf9acad9d92e3b8a6ef2309efa515.